tests/lib/reset.sh

```diff
@@ -44,7 +59,11 @@ reset_classic() {
         systemctl start snapd.socket
 
         # wait for snapd listening
-        while ! printf "GET / HTTP/1.0\r\n\r\n" | nc -U -q 1 /run/snapd.socket; do sleep 0.5; done
+        EXTRA_NC_ARGS="-q 1"
+        if [[ "$SPREAD_SYSTEM" = fedora-* ]]; then
+            EXTRA_NC_ARGS=""
```

Comment:
Ok, I have an replacement for this coming with another PR. Would prefer to keep this as is and fix it properly with a followup PR.